molgenis-data-postgresql/src/test/java/org/molgenis/data/postgresql/identifier/EntityTypeRegistryImplTest.java

```diff
@@ -47,15 +50,46 @@ public void testGetEntityTypeDescription() throws Exception
 	}
 
 	@Test
-	public void testGetEntityTypeDescriptionJunctionTable() throws Exception
+	public void testUnregisterEntityType()
 	{
-		String entityTypeId = "entityTypeId";
-		EntityType entityType = mock(EntityType.class);
-		when(entityType.getAllAttributes()).thenReturn(emptyList());
-		when(entityType.getId()).thenReturn(entityTypeId);
+		EntityType entityType = prepareMockEntityTypeForRegistering();
+
 		entityTypeRegistryImpl.registerEntityType(entityType);
-		String tableName = "entityTypeId#c34894ba_mrefattr";
-		assertEquals(entityTypeRegistryImpl.getEntityTypeDescription(tableName),
-				EntityTypeDescription.create(entityTypeId, ImmutableMap.of()));
+
+		assertNotNull(entityTypeRegistryImpl.getEntityTypeDescription("org_molgenis_test_many_packages_deep#2409e1c6"));
```

Comment:
Probably a bit crisper to check that the entity type id in the EntityTypeDescription is correct?